ingestion-data/transfer-config/HLSL30.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
should we remove transfer config for HLSL30 & HLSS30? I think these were predecessors to the hls-ej-reprocessed collections?

Reply:
No action requested on HLS ^, just for discussion

Reply:
yup will do

Reply:
So those came from the following files in discovery items:

- HLSL30.002-ida.json
- HLSL30.002-maria.json
- HLSS30.002-ida.json
- HLSS30.002-maria.json

I'm assuming they were never used since we don't have collection ids `HLSL30` or `HLSS30`

Reply:
> I'm assuming they were never used since we don't have collection ids HLSL30 or HLSS30

I think they were used but were quickly replaced because we had to reprocess the data:

- https://github.com/NASA-IMPACT/veda-data-pipelines/issues/147
- https://github.com/NASA-IMPACT/veda-data-pipelines/issues/89

Reply:
https://github.com/NASA-IMPACT/veda-data/issues/110 carries this discussion forward